apps/site/components/Downloads/DownloadReleasesTable/index.tsx

```diff
@@ -0,0 +1,58 @@
+import Badge from '@node-core/ui-components/Common/Badge';
+import { getTranslations } from 'next-intl/server';
+import type { FC } from 'react';
+
+import FormattedTime from '@/components/Common/FormattedTime';
+import DetailsButton from '@/components/Downloads/DownloadReleasesTable/DetailsButton';
+import getReleaseData from '@/next-data/releaseData';
+
+// This is a React Async Server Component
+// Note that Hooks cannot be used in a RSC async component
+// Async Components do not get re-rendered at all.
+const DownloadReleasesTable: FC = async () => {
```

Comment:
I think these comments are superfluous now that react devs know that an async component is a react component. 
If I remember correctly, this comment was introduced when RSCs were introduced.

```suggestion
```